addon-test-support/load.js

```diff
@@ -19,4 +22,69 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes("loadBalance")) {
+    return;
+  }
+  qunit.begin(function() {
+    return new Promise(function(resolve) {
+      Testem.on('testem:next-module-response', function getTestModule(moduleName) {
```

Comment:
Not a huge thing, and you can totally ignore this, but what about using something like [`p-event`](https://www.npmjs.com/package/p-event) here?